.azure-pipelines/build-frameworks-and-docs-1ES.yml

```diff
@@ -97,4 +97,37 @@ extends:
             echo $VERSION > $(Build.ArtifactStagingDirectory)/Info/VERSION
             echo $BUILD_SOURCEVERSION > $(Build.ArtifactStagingDirectory)/Info/COMMIT
             Scripts/get-release-notes.sh $VERSION > $(Build.ArtifactStagingDirectory)/Info/RELEASE.md
-          displayName: 'Collect Version Information'
\ No newline at end of file
+          displayName: 'Collect Version Information'
+
+    - stage: APIScan
+      dependsOn: Stage
+      pool:
+        name: 1ES-PT-Windows-2022
+        os: windows
+      variables:
+        "agent.source.skip": true
+      jobs:
+      - job: APIScan
+        steps:
+        - task: DownloadPipelineArtifact@2
+          displayName: Download Build Artifacts for APIScan
+          inputs:
+            artifactName: Release
+            targetPath: '$(Agent.BuildDirectory)/Release'
+        - task: AzureKeyVault@2
+          inputs:
+            azureSubscription: 'AC - Dev Infra & Build Pool'
+            KeyVaultName: 'mobile-center-sdk'
+            SecretsFilter: 'appcenter-sdk-managed-identity-clientid'
+            RunAsPreJob: false
+        - task: APIScan@2
+          displayName: 'Run APIScan'
+          inputs:
+            softwareFolder: '$(Agent.BuildDirectory)\Release'
+            softwareName: 'appcenter-sdk-apple'
+            softwareVersionNum: Latest
```

Comment:
This is imho wrong - see https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki/25351/APIScan-step-by-step-guide-to-setting-up-a-Pipeline

```
  inputs:
    softwareFolder: $(Agent.TempDirectory)
    softwareName: 'MySoftware'
    softwareVersionNum: '$(Build.BuildId)'
    isLargeApp: false
    toolVersion: 'Latest'
```

Reply:
fixed